libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/changeAttributeFilterModeHandler.ts

```diff
@@ -0,0 +1,42 @@
+// (C) 2023 GoodData Corporation
```

Comment:
This command handle to update data of dashboard level, please consider to move it to dashboard folder!

Reply:
Updated. Thanks!